src/theme/basic/templates/plugin-column-sort.tpl.html

```diff
@@ -1,5 +1,6 @@
 <ng-template key="plugin-column-sort.tpl.html" let-$columnSort>
-    <span class="q-grid-icon q-grid-asc">&uarr;</span>
-    <span class="q-grid-icon q-grid-desc">&darr;</span>
+    <i class="material-icons q-grid-icon q-grid-asc">arrow_upward</i>
```

Comment:
need to you UTF symbols no material is allowed in this theme

Reply:
fixed in the last commit